docs/source/cleanlab/datalab/internal/issue_manager/index.rst

```diff
@@ -17,6 +17,7 @@ These are the issue managers that Datalab has registered.
     duplicate
     noniid
     imbalance
+    underperforming_group
```

Comment:
confirm this is not part of the default set of issues that datalab detects automatically, right?

Please open GH issue to track adding Underperforming Group to that default set in a future PR.

Reply:
Yes, the underperforming_group issue type is not part of the default set of issues.